examples/src/h2c/server.rs

```diff
@@ -100,8 +98,12 @@ mod h2c {
             std::task::Poll::Ready(Ok(()))
         }
 
-        fn call(&mut self, mut req: hyper::Request<Incoming>) -> Self::Future {
-            let svc = self.s.clone();
+        fn call(&mut self, req: hyper::Request<Incoming>) -> Self::Future {
+            let mut req = req.map(tonic::body::boxed);
+            let svc = self
+                .s
+                .clone()
+                .map_request(|req: Request<Incoming>| req.map(tonic::body::boxed));
```

Comment:
Does this really need to specify the type of `req`?

Reply:
The body type seems not to be needed. Fixed it.